galaxy_ng/app/migrations/0008_rename_default_repo.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 2.2.15 on 2020-09-17 16:55
+
+import contextlib
+
+from django.db import migrations
+
+def rename_default_repo(apps, schema_editor):
+    AnsibleRepository = apps.get_model('ansible', 'AnsibleRepository')
+    AnsibleDistribution = apps.get_model('ansible', 'AnsibleDistribution')
+    db_alias = schema_editor.connection.alias
+
+    old_name = 'automation-hub'
+    new_name = 'published'
+
+    with contextlib.suppress(AnsibleRepository.DoesNotExist, AnsibleDistribution.DoesNotExist):
+        repo = AnsibleRepository.objects.using(db_alias).get(name=old_name)
+        distro = AnsibleDistribution.objects.using(db_alias).get(base_path=old_name)
+
+        repo.name = new_name
```

Comment:
This fails if the "published" distro/repo already exists. Not sure if that's a problem or not.

Reply:
@newswangerd we probably should skip the migration if the user has created their own "published" repo/distro. Updated.

Reply:
I think that's a good plan